parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -454,6 +465,17 @@ try
         Write-Log "Start calico installation"
         Start-InstallCalico -RootDir "c:\" -KubeServiceCIDR $global:KubeServiceCIDR -KubeDnsServiceIp $KubeDnsServiceIp
     }
+    
+    $RebootNeeded = $false

```

Comment:
Prefer below code to reduce the length of customdata (cse)
1. always import `. c:\AzureData\windows\nvidiagpudriverfunc.ps1` with other cse scripts together
2. $RebootNeeded = Start-InstallGPUDriver --enable-install $global:ConfigGPUDriverIfNeeded (Log it in Start-InstallGPUDriver when $global:ConfigGPUDriverIfNeeded is true and also log it if it is skipped $global:ConfigGPUDriverIfNeeded is false